APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+Setting an appropriate level of pay for certain work is a difficult task, since a range of factors influences what people think is “fair”. This includes, to name just a few considerations,
+
+* external factors like the cost of living in a certain place or country, 
+* if people are paid as independent contractors (and thus have to factor vacation, sick time, medical insurance etc. into an hourly rate) or as a subaward to e.g. an academic institution (which usually set rates that include overheads and fringe benefits which are then provided  to the employee),
+* What monetary value one puts on experience (e.g. an undergraduate student vs. an experienced software developer).
+
+`NEP 48 <https://numpy.org/neps/nep-0048-spending-project-funds.html>`_ has extensive discussion and a good list of further references on this topic.
+
+We recognize though that the approach of leaving the budget entirely up to project members also has downsides: In particular, less experienced people may be tempted to suggest work packages with a compensation that is unfairly low for the work required to increase their perceived chances to obtain funding. We place trust in the community to recognize these short-comings of the process and to advise and mentor each other to propose realistic budgets that allow the proposed work to be completed with a fair rate of pay. As the diversity of proposers increases, we recognize that this approach will likely need to be modified and potentially be made more prescriptive. This may come in the form of a future APE or more incremental changes as experience grows.
+
+Defining fundable activities and projects
+-----------------------------------------
+We define three separate categories of expense, by value:
+
+* Up to $2000 (US);
+* $2000 to $10000 (US);
+* More than $10000 (US).
+
+Items in the first category — up to $2000 — may be approved by majority vote of the Finance Committee.
+
+Items in the second category — up to $10000 — can be approved by majority vote of the Finance Committee. If, in the judgment of one or more Finance Committee members, the item would not receive widespread support, the Committee will communicate these concerns to the CoCo. The CoCo may provide a decision, or may request that the Finance Committee organize a vote among the Astropy membership to approve the expense.
+
+Items in the third category — more than $10000 — should follow approved Project decision making methods from APE0, or a vote among the Astropy voting membership unless specifically allocated in a grant to a specific person or organization already; in the latter case the Finance Committee is tasked to ensure it is spend according to the provisions of the grant. The Finance Committee will periodically solicit suggestions from the community for projects and other major expenses and coordinate a selection process.
```

Comment:
In what situations is are APE0 processes followed vs. a voting membership vote?

Reply:
I would say it's up to Coco and finance committee how exactly that is done in each case. [APE0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst) has several sections that might be relevant:

- (in the section on the role of the Coco): "Seek consensus among the Astropy community, particularly before acting in a formal capacity,"
- (in the section on the Coco): "To use its authority, the Coordination Committee must first attempt to reach a consensus. If a consensus cannot be reached, the Committee votes."
- (in the section on voting members): "Voting Members may and should participate in formal votes on: [...] Other matters for which the Coordination Committee believes a vote is appropriate."

So my goal here was write it in such a way that voting by voting members is the "normal", but not conflict with APE0, because APE0 aims to describe how any and all decisions are made in Astropy and this APE does not have the intent (nor the power) to overrule APE0.